common/djangoapps/util/milestones_helpers.py

```diff
@@ -413,3 +414,16 @@ def add_user_milestone(user, milestone):
         return None
     from milestones import api as milestones_api
     return milestones_api.add_user_milestone(user, milestone)
+
+
+def get_milestones_service():
```

Comment:
`get_services` is sufficient as the name, since it's namespaced to the module that it lives in.  So callers would call `milestones_helpers.get_service()`.